### PR TITLE
Alphabetized, fixed spelling error, clarifying event descriptions. Polish

### PR DESCRIPTION
**Pull request by KirbyDaMaster**
## About The Pull Request

Alphabetized several long lists of strings so its easier for us to look through them, just code polish, nothing the players would see. 

Fixed some minor spelling errors as well.

Clarified door bolt state to be less ambiguous in the door wiring gui.
Originally it would say the door bolts have fallen, and the door bolts "Look up". i dont know about you but that was very not clear for me to read. Like where are the bolts? In the door or the frame? Arnt there bolts on top and bottom? Just didn't make sense to me.

Now it says "Have engaged!" & "Have disengaged"
hopefully that makes the state clearer at a glance.

I also added a small handful of funny texts to some string files. See changelog

## Why It's Good For The Game

Well, who doesn't like a bit of polish? Just makes the game a little easier for people.
Also funny text funny text.

## Changelog
spelling: improves spelling and adds more flavortext

### CODE REVIEW

File: strings/steve.json [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
         "I dunno about you guys, but IDs and PDAs being separate is clunky as fuck. Maybe we should merge them into a chip in our arms? That way they can't be stolen easily.",
+        "I'm thinking we should roll out a git repository for our research under the AGPLv3 license so that we can share it among the other stations freely.",
         "Why the fuck aren't we just making every pair of shoes into galoshes? We have the technology."
     ],
 
     "ballmer_windows_me_msg": [
-        "Yo man, what if, we like, uh, put a webserver that's automatically turned on with default admin passwords into every PDA?",
-        "So like, you know how we separate our codebase from the master copy that runs on our consumer boxes? What if we merged the two and undid the separation between codebase and server?",
-        "Dude, radical idea: H.O.N.K mechs but with no bananium required.",
+		"Who keeps commiting to master?",
```
Comment (Time-Green) [inline strings/steve.json:11]: We'll find them. Also might just be git but indentation seems wrong here

File: strings/wisdoms.txt [hunk 2]
```diff
@@ -424,7 +411,7 @@ Your love life will be... interesting.
 Your lover will never wish to leave you.
 Your lucky color has faded.
 Your lucky number has been disconnected.
-Your lucky number is 3552664958674928.  Watch for it everywhere.
```
Comment (Time-Green) [inline strings/wisdoms.txt:427]: aw

**Comment by Fikou** [comment]
🎏

**Review by Time-Green — COMMENTED** [review]
Dear god this is a lot. Nice work though!

**Comment by Time-Green** [comment]
You might also want to add a changelog, like this:
:cl: 
spelling: improves spelling and adds more flavortext
/:cl:
That way players can see something changed, and you'll get to take your rightful place in the glorious changelog

File: strings/steve.json (hunk 1 above)
Comment (KirbyDaMaster) [reply strings/steve.json:11]: not sure how that happened, was not intentional

File: strings/greek_letters.txt [hunk 3]
```diff
@@ -1,24 +1,24 @@
 Alpha
 Beta
-Gamma
+Chi
```
[the remark this answers is not in the document]
Comment (KirbyDaMaster) [reply strings/greek_letters.txt:3]: Honestly was on clean up autopilot at this point. We can ignore this change

File: strings/steve.json (hunk 1 above)
Comment (KirbyDaMaster) [reply strings/steve.json:11]: thats weird, when i click on the actual file and look at it it doesnt show that weird indent.

Comment (Time-Green) [reply strings/steve.json:11]: Might be a git bug then, it's fine

File: .gitignore [hunk 4]
```diff
@@ -207,3 +207,5 @@ aux*.pdb
 
 # Screenshot tests
 /artifacts
+.vs/slnx.sqlite
+.vs/VSWorkspaceState.json
```
Comment (LemonInTheDark) [inline .gitignore:211]: ?

Comment (DaedalusGame) [reply .gitignore]: fixed

File: strings/steve.json (hunk 1 above)
Comment (MrStonedOne) [reply strings/steve.json:11]: This file is space indented, kirby's additions are tabs

**Pull request merged**